app/imports/schemas/TestForm.js

```diff
@@ -1,23 +1,40 @@
-export const FormSchema = new SimpleSchema({
+/* globals _ */
+import { Upload } from '/imports/schemas/Upload.js';
+
+const Schema = new SimpleSchema({
   name: {
     type: String,
   },
 
-  category: {
-    type: String,
-    allowedValues: [
-      'news',
-      'image',
-      'video',
-    ],
+  idCard: {
+    type: Upload,
+    // This makes the field 'required'
+    autoValue() {
+      if (_.isEmpty(this.value.file)) {
+        this.unset();
+      }
+    },
   },
 
-  publishedDate: {
-    type: Date,
+  cv: {
+    type: Upload,
   },
 
-  published: {
-    type: Boolean,
-    optional: true,
+  photos: {
+    type: [ Upload ],
+    maxCount: 2,
   },
+  
+  docs: {
+    type: [ Upload ],
+    // This makes minCount work
+    autoValue() {
+      return _.filter(this.value, (val) => !_.isEmpty(val.file));
+    },
+    minCount: 1,
+    maxCount: 2,
```

Comment:
the important part is commented, the rest is simpleschema basics, idk if u r not familiar with it. but i feel @mikeyrayvon is and would understand most of the schema well